Cargo.lock

```diff
@@ -1540,6 +1571,23 @@ version = "0.3.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "9b919933a397b79c37e33b77bb2aa3dc8eb6e165ad809e58ff75bc7db2e34574"
 
+[[package]]
+name = "governor"
+version = "0.4.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "19775995ee20209163239355bc3ad2f33f83da35d9ef72dea26e5af753552c87"
+dependencies = [
+ "dashmap",
+ "futures",
+ "futures-timer",
+ "no-std-compat",
+ "nonzero_ext",
+ "parking_lot",
+ "quanta",
+ "rand",
+ "smallvec",
+]
```

Comment:
I couldn't find a way to disable the features of governor via actix-governor (that uses governor's default features). It is also unclear to me if we can disable them at all: the guide for governor [mentions](https://docs.rs/governor/latest/governor/_guide/index.html#constructing-a-keyed-rate-limiter) that it uses dashmap by default in the "std mode" but it is unclear what to do to change this and the impact it might have.